samples/glean/metrics.yaml

```diff
@@ -1,22 +1,17 @@
-# This file contains a definition of the scalar probes that are recorded in Telemetry.
```

Comment:
Yeah, this comment was weird :) Can we still retain some similar comment at the top of the file?

Reply:
I've put a comment back in here.